src/client/csfle/client_encryption.rs

```diff
@@ -331,24 +483,40 @@ pub enum MasterKey {
 // }
 
 /// The options for explicit encryption.
-#[derive(Debug, Clone, TypedBuilder)]
-#[builder(field_defaults(setter(into)))]
+#[derive(Debug, Clone)]
 #[non_exhaustive]
-pub struct EncryptOptions {
-    /// The key to use.
-    pub key: EncryptKey,
-    /// The encryption algorithm.
-    ///
-    /// To insert or query with an "Indexed" encrypted payload, use a `Client` configured with
-    /// `AutoEncryptionOptions`. `AutoEncryptionOptions.bypass_query_analysis may be true.
-    /// `AutoEncryptionOptions.bypass_auto_encryption` must be false.
-    pub algorithm: Algorithm,
-    /// The contention factor.
-    #[builder(default)]
-    pub contention_factor: Option<i64>,
-    /// The query type.
-    #[builder(default)]
-    pub query_type: Option<String>,
+pub(crate) struct EncryptOptions {
```

Comment:
With the action pattern, there's no immediate reason for this to be a public type.  If it turns out that there's a need for it adding it back in will be semver-compatible 🙂